proto/greptime/v1/ddl.proto

```diff
@@ -60,6 +62,25 @@ message DropFlowExpr {
   bool drop_if_exists = 5;
 }
 
+// Create a view
+message CreateViewExpr {
+  string catalog_name = 1;
+  string schema_name = 2;
+  string view_name = 3;
+  bytes logical_plan = 4;
+  bool create_if_not_exists = 5;
+  bool or_replace = 6;
```

Comment:
Is there no `desc` field?

Reply:
No,it doesn't have a description field, but maybe we can support `with options`.